src/main/java/com/strandls/esmodule/ApiConstants.java

```diff
@@ -47,7 +47,8 @@ private ApiConstants() {}
 	public static final String FORCEUPDATE = "forceupdateindex";
 	public static final String FETCHINDEX="fetchindex";
 	public static final String USERINFO = "/userInfo";
-	
+	public static final String UPLOADERSINFO="/uploadersInfo";
```

Comment:
please put proper spacing, format the code using cntrl+shift+f

Reply:
applied proper spacing